spec/factories/supplejack_api_top_collection_metrics.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
```

Comment:
This looks like an empty version of the one below?

Reply:
gone.